src/audio/copier/copier.c

```diff
@@ -36,6 +36,7 @@
 #include <errno.h>
 #include <stddef.h>
 #include <stdint.h>
+#include <sof/audio/host_copier.h>
```

Comment:
@btian1 I think you should squash the first 3 patches so it is easy to see why you need the common header and the hd pointer in the copier_data

Reply:
@ranj063 , do you mean combine first 3 patches to one? with current patch, people may easy to start and access this patch, do you think so?